docs/cron-workflows.md

```diff
@@ -103,6 +104,33 @@ For example, with timezone set at `America/Los_Angeles`, we have daylight saving
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+### Stopping a `CronWorkflow`
+
+> v3.6 and after
+
+You can stop the execution of new workflows from a `CronWorkflow` based on the number of previous successes or failures. The spec field `stopStrategy` accepts a `condition` which is an [expression](variables.md#expression) that stops triggering new workflows when it evaluates to true.
+
+Keywords allowed for stopping conditions are `failed` and `succeeded`:
+
+- `failed` - counts how many times the child workflows failed
+- `succeeded` - counts how many times the child workflows succeeded
+
+For example, if you want a workflow to be executed only once you can define the spec as:
+
+```yaml
+stopStrategy:
+  condition: "succeeded >= 1"
+```
+
+You can also stop scheduling new workflows after three failures with:
+
+```yaml
+stopStrategy:
+  condition: "failed >= 3"
+```
+
+**NOTE:** Depending on the time it takes to schedule and run a workflow, it is possible that the total number of executions of a `CronWorkflow` is higher than the configured max executions. For example, if you configure the `CronWorkflow` to execute every minute with a stopping condition: `succeeded >= 1` and the `Workflow` executes in 90 seconds, the `CronWorkflow` will be completed with 2 executions, because when the stopping condition is achieved, there is another workflow running and the controller is not canceling active workflows. For that reason, it is advisable to have conditions like `succeeded  >= 1` instead of `succeeded == 1`.
```

Comment:
This should use a material admonition (see also #12561), and really should be a warning as it's pretty important.

```suggestion
!!! Warning Scheduling vs. Completions
     Depending on the time it takes to schedule and run a workflow, the number of completions can exceed the configured maximum.
     
     For example, say you configure a `CronWorkflow` to schedule every minute (`* * * * *`) and stop after one success (`succeeded >= 1`).
     If the `Workflow` takes 90 seconds to run, the `CronWorkflow` will actually stop after two completions.
     This is because when the stopping condition is achieved, there is _already_ another `Workflow` running.
     For that reason, prefer conditions like `succeeded  >= 1` over `succeeded == 1`.
```
- prefer 1 sentence per line of markdown, per [style guide](https://argo-workflows.readthedocs.io/en/latest/doc-changes/).
- simpler and more direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
  - "it is possible that the total" can be simplified
  - "is higher than" -> "exceed", "max executions" -> "maximum"
  
- be consistent with terminology and numerics
  - "schedule" over "execute", same as above (and [previous suggestions](https://github.com/argoproj/argo-workflows/pull/12305#discussion_r1501842596))
  - "executions" -> "completions" -- as "complete" is an existing term the docs already use
  - "will be completed" -> "will stop" -- as this is `stopStrategy` and this section is on "stopping"
  - show both the condition and schedule in their respective YAML notation
  - "two" instead of "2" as you have "three" and "one" previously
  - "`Workflow`" to be consistent with in this note
  
- re-word the last sentence for simplicity and active voice, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)

- "the controller" is an architectural implementation detail that doesn't need to be stated here. that running workflows are not cancelled is also implied by the very nature of this note, so I don't think that's necessary either.